src/components/ColourModeSwitch.tsx

```diff
@@ -0,0 +1,21 @@
+import { Accessor, Setter, Show } from "solid-js";
+
+export default function ColourModeSwitch({
+  getter,
+  setter,
+}: {
+  getter: Accessor<boolean>;
+  setter: Setter<boolean>;
+}) {
```

Comment:
Create `interface Properties` and use that instead.

Reply:
Good idea.